acceptance/tests/ingress-gateway/ingress_gateway_namespaces_test.go

```diff
@@ -44,13 +44,19 @@ func TestIngressGatewaySingleNamespace(t *testing.T) {
 
 			// Install the Helm chart without the ingress gateway first
 			// so that we can create the namespace for it.
+			igName := "ingress-gateway"
```

Comment:
Does this change need to be undone? Are you going to run into issues installing without the Consul namespace?

Reply:
This test only runs when namespaces are enabled

Reply:
Right, namespaces are enabled, but does the particular namespace the ingress gateway is being registered in exist? I thought that's why the previous code had you install, create the particular namespace, then upgrade.

Reply:
aaha! i ignored that comment entirely lol. i have updated the endpoints controller to now create the gateway namespace if it does not already exist

Reply:
Sweet! Then it's good to go.

Reply:
Oh, actually, there is still test code that creates a Consul namespace in the secure case. I believe it's a no-op and can be removed.

Reply:
done..good catch